### PR TITLE
issue 594 define schema for JSONField

### PR DESCRIPTION
**Pull request by r-peschke**
Defined a specific schema for JSON field and changed some tests

### CODE REVIEW

**Review by FinnStutzenstein — APPROVED** [review]
Tested with the client. Works

File: openslides_backend/models/fields.py [hunk 1]
```diff
@@ -100,7 +100,12 @@ def get_schema(self) -> Schema:
 
 
 class JSONField(TextField):
```
Comment (jsangmeister) [inline openslides_backend/models/fields.py:102]: I think this no longer should inherit from `TextField`, right? I think the initial thought was that it should represent the json-encoded string, but this is obsolet now.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/models/fields.py (hunk 1 above)
Comment (r-peschke) [reply openslides_backend/models/fields.py]: Yes and done

File: tests/system/action/projector/test_project.py [hunk 2]
```diff
@@ -135,49 +135,10 @@ def test_project_not_unique_ids(self) -> None:
...
-                "options": "",
+                "options": {},
                 "stable": False,
                 "type": "test",
             },
         )
         self.assert_status_code(response, 400)
         assert "data.ids must contain unique items" in response.data.decode()
-
-    def test_meeting_as_content_object_ok(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/projector/test_project.py:146]: I don't think you meant to remove these tests, right? :D

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**